ADAL/src/validation/ADAuthorityValidation.m

```diff
@@ -185,6 +185,7 @@ - (void)checkAuthority:(ADRequestParameters*)requestParams
     {
         // Validate AAD authority
         [self validateAADAuthority:authorityURL
+                 validateAuthority:validateAuthority
```

Comment:
Tiny nit/question: should we name the new parameter to something like "shouldValidateAuthority"? Just because the method is also named a variation of "validateAuthority".

Reply:
great suggestion, addressed :)